examples/workflow-events/src/main/java/io/littlehorse/examples/WorkflowEventsExample.java

```diff
@@ -0,0 +1,90 @@
+package io.littlehorse.examples;
+
+import io.littlehorse.sdk.common.config.LHConfig;
+import io.littlehorse.sdk.common.proto.Comparator;
+import io.littlehorse.sdk.common.proto.VariableMutationType;
+import io.littlehorse.sdk.common.proto.VariableType;
+import io.littlehorse.sdk.wfsdk.TaskNodeOutput;
+import io.littlehorse.sdk.wfsdk.WfRunVariable;
+import io.littlehorse.sdk.wfsdk.Workflow;
+import io.littlehorse.sdk.wfsdk.internal.WorkflowImpl;
+import io.littlehorse.sdk.worker.LHTaskWorker;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Properties;
+
+public class WorkflowEventsExample {
+
+    public static Workflow getWorkflow() {
+        return new WorkflowImpl(
+            "example-workflow-events",
+            wf -> {
+                WfRunVariable numberOfIterations = wf.addVariable("number-of-iterations", VariableType.DOUBLE);
```

Comment:
some documentation over this example, i'tll help to understand, or refactor the code